tasks/publish-index-image/README.md

```diff
@@ -4,16 +4,19 @@ Publish a built FBC index image using skopeo
 
 ## Parameters
 
-| Name | Description | Optional | Default value |
-|------|-------------|----------|---------------|
-| targetIndex | targetIndex signing image | No | |
-| inputDataFile | Optional file to read data from | No | |
-| retries | Number of skopeo retries | Yes | "0" |
-| publishingCredentials | The credentials used to access the registries | Yes | "fbc-publishing-credentials" |
-| requestUpdateTimeout | Max seconds waiting for the status update | Yes | 360 |
+| Name           | Description                                                             | Optional | Default value |
+|----------------|-------------------------------------------------------------------------|----------|---------------|
+| dataPath       | Path to the JSON string of the merged data to use in the data workspace | Yes      | "data.json"   |
```

Comment:
I would suggest removing the `""` here and also for `retries` - at least from what I've seen, we usually don't use those here and it's implied that everything is a string.